src/SamplesApp/UITests.Shared/UITests.Shared.projitems

```diff
@@ -4461,6 +4466,12 @@
     <Compile Update="$(MSBuildThisFileDirectory)Windows_UI_Xaml_Controls\ToolTip\TextOnlyToolTipSample.xaml.cs">
       <DependentUpon>TextOnlyToolTipSample.xaml</DependentUpon>
     </Compile>
+    <Compile Update="C:\P\Uno\Uno.UI\src\SamplesApp\UITests.Shared\Windows_UI_Xaml_Controls\ListView\ListView_VirtualizePanelAdaptaterIdCache.xaml.cs">
```

Comment:
should not be an absolute path (`C:\etc`) here - this is coming from a VS bug, you can fix it manually